.github/workflows/owasp.yml

```diff
@@ -0,0 +1,63 @@
+
+name: Daily scan for dependencies
+on:
+  schedule:
+    - cron: "22 3 * * *"
+  workflow_dispatch:
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  check-dependencies-adot-java:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 17
+      - name: Build snapshot with Gradle
+        uses: gradle/gradle-build-action@v2
+        with:
+          name: adot-dependencies
+          arguments: ":otelagent:dependencyCheckAnalyze"
+
+  check-dependencies-otel-java:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          name: "open-telemetry/opentelemetry-java-instrumentation"
+          fetch-depth: 0
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 17
+      - name: Build snapshot with Gradle
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: ":javaagent:dependencyCheckAnalyze"
+      - name: Upload report
+        if: always()
```

Comment:
Does this require enclosing in `${{ }}`

Reply:
https://docs.github.com/en/actions/learn-github-actions/expressions

Docs say that it is optional, but for consistency I will add it. good call. thanks!